plugins/elantp/fu-elantp-hid-device.c

```diff
@@ -166,14 +166,18 @@ fu_elantp_hid_device_setup (FuDevice *device, GError **error)
 	version = fu_common_version_from_uint16 (fwver, FWUPD_VERSION_FORMAT_HEX);
 	fu_device_set_version (device, version);
 
-	/* get EAP firmware version */
+	/* get IAP firmware version */
 	if (!fu_elantp_hid_device_read_cmd (self,
 					    self->pattern == 0 ? ETP_CMD_I2C_IAP_VERSION : ETP_CMD_I2C_IAP_VERSION_2,
 					    buf, sizeof(buf), error)) {
 		g_prefix_error (error, "failed to read bootloader version: ");
 		return FALSE;
 	}
```

Comment:
Is it possible to change "fwver" to "iap_ver" for fear of confusion with the real TP Firmware version.

Reply:
Good idea, fixed.